src/tasks/initiate_new_auction.ts

```diff
@@ -84,6 +90,29 @@ const initiateAuction: () => void = () => {
       );
 
       console.log("Using EasyAuction deployed to:", easyAuction.address);
+      if (
+        taskArgs.allowListManager !=
+        "0x0000000000000000000000000000000000000000"
+      ) {
+        const allowListManager = await hardhatRuntime.ethers.getContractAt(
+          "AllowListVerifier",
+          taskArgs.allowListManager,
+        );
+        if (
+          allowListManager.supportsInterface(
```

Comment:
Here, in the initiate auction script, EIP165 is handy, though it feels like overkill for what I wanna check.